src/IntroScreen.cpp

```diff
@@ -62,59 +66,87 @@ IntroScreen::IntroScreen(ResourceManager& resmgr, sf::RenderTarget& target)
     : Screen(resmgr, target)
 {
     
-    if (auto temp = _resources.load<sf::Font>("fonts/hobo.ttf"); 
+    if (auto temp = _resources.load<sf::Font>("fonts/pricedown.ttf"); 
         temp.has_value())
     {
         _font = *temp;
     }
     else
     {
-        throw std::runtime_error("hobo.ttf could not be loaded!");
+        throw std::runtime_error("fonts/pricedown.ttf could not be loaded!");
     }
 
     //
     // This textobj is the title text
     //
     auto textobj = std::make_shared<sf::Text>(
-                    "The Tommy Tucson\nVideo Game", _font);
+                    "Grand Theft Hobo", _font);
 
     textobj->setCharacterSize(70);
     textobj->setFillColor(sf::Color(255, 215, 9));
 
+    textobj->setOutlineColor(sf::Color(0,0,0,255));
+    textobj->setOutlineThickness(5);
+
     //
     // Get coords for title from window size. (i.e. xpos)
     //
     auto winWidth       = _window.getSize().x;
-
+    auto winHeight      = _window.getSize().y;
     auto titleWidth     = textobj->getLocalBounds().width;
-    auto titleXpos      = (winWidth - titleWidth) - 100.f;
+    auto titleXpos      = (winWidth - titleWidth) - 25.f;
 
     //
     // Set coords for title text
     //
     textobj->setPosition(titleXpos - 20, 10);
 
     //
-    // Load the scrolling Tom image.
+    // Load background images
     //
-    auto bgt = _resources.load<sf::Texture>("images/tommy-1.png");
+    for(int i = 0; i < 8; i++) {
 
-    if (!bgt)
-    {
-        throw std::runtime_error("tommy-1.png could not be loaded!");
-    }
+        //
+        // Generate file name strings.
+        //
+        std::string prefix  = "images/ttvg-intro-screen-";
+        std::string index   = std::to_string(i+1);
+        std::string suffix  = ".png";
```

Comment:
Prefer to use `const` objects since compilers can do nifty optimizations if they know the variables will never change.

Also, just a pointer (ha!): We're using the `fmt` library in this project, so you could construct this string like so:
```
const auto filename = fmt::format("images/ttvg-intro-screen-{}.png", i+1);
auto bgt = _resources.load<sf::Texture>(filename);
```
Which is more readable and efficient.